File: module5/test-module5/test-m5/src/models/student.ts
export class Student {
  private id: number;
  private name: string;
  private nameGroup: number;
  private topic: string;
  private teacher: number;
  private email: string;
  private phone: string;

  constructor(id: number, name: string, nameGroup: number, topic: string, teacher: number, email: string, phone: string) {
    this.id = id;
    this.name = name;
    this.nameGroup = nameGroup;
    this.topic = topic;
    this.teacher = teacher;
    this.email = email;
    this.phone = phone;
  }
}

File: module5/test-module5/test-spring/src/main/java/com/student/models/Student.java
package com.student.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@Table(name = "student")
public class Student {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private String name;

    private Integer nameGroup;

    private String topic;

    private Integer teacher;

    private String email;

    private String phone;

}
